generate_file_tree.sh

```
@@ -0,0 +1,43 @@
+TARGET_FOLDER="docs"
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)